src/audio/src/src.c

```diff
@@ -815,52 +832,6 @@ static int src_trigger(struct comp_dev *dev, int cmd)
 	return comp_set_state(dev, cmd);
 }
 
-static int src_get_copy_limits(struct comp_data *cd,
-			       const struct comp_buffer __sparse_cache *source,
-			       const struct comp_buffer __sparse_cache *sink)
-{
```

Comment:
please keep this function where it was if possible

Reply:
updated